### PR TITLE
Updating devfile / contribute section to the workspaces.openshift.com

### PR DESCRIPTION
**Pull request by ibuziuk**
Signed-off-by: Ilya Buziuk <ibuziuk@redhat.com>

Try it on Developer Sandbox:

[Contribute](https://workspaces.openshift.com/f?url=https://github.com/ibuziuk/che-website/tree/migrating_to_workspaces.openshift.com)

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,7 +3,9 @@ https://www.eclipse.org/che website mirror
 
 ## Contribute with the devfile
 
-Use the factory url https://che.openshift.io/f?url=https://github.com/eclipse/che-website.
+[![Contribute](https://raw.githubusercontent.com/redhat-developer-demos/quarkus-reactjs-postit-app/master/factory-contribute.svg)](https://workspaces.openshift.com/f?url=https://github.com/eclipse/che-website)
```
Comment (sunix) [inline README.md:6]: In my opinion you have to tell the user to register first, otherwise it won't load the workspace the first time

Comment (ibuziuk) [reply README.md:6]: yeah, but user will be redirected to the developer-sandbox if there is no account

Comment (sunix) [reply README.md:6]: I think it is not straightforward. so we would have people going through the registration but what are the chance to see them back clicking on the factory link ?

File: devfile.yaml [hunk 2]
```diff
@@ -2,17 +2,13 @@
...
 components:
-  - alias: git
-    type: dockerimage
-    image: sunix/git-devtools
-    mountSources: true
-    memoryLimit: 256M
-
   - alias: php
     type: dockerimage
-    image: eclipse/php
```
Comment (ibuziuk) [inline devfile.yaml:15]: @sunix could you please migrate the image from docker to quay?

File: devfile.yaml [hunk 3]
```diff
@@ -2,17 +2,13 @@
...
 metadata:
    generateName: che-website-
-
+attributes:
+  persistVolumes: 'false'
+  
 components:
-  - alias: git
-    type: dockerimage
-    image: sunix/git-devtools
```
Comment (ibuziuk) [inline devfile.yaml:9]: @sunix can we make this image official eclipse on the quay?

**Comment by benoitf** [comment]
closing as outdated

**Pull request closed without merging**